site/src/index.js

```diff
@@ -60,3 +74,19 @@ r.add('delete', '/api/internal/tokens', tokensDelete)
 
 r.add('all', '*', notFound)
 addEventListener('fetch', r.listen.bind(r))
+
+// Cron jobs
+addEventListener('scheduled', (event) =>
+  event.waitUntil(
+    (async () => {
+      await timed(updateUserMetrics, 'CRON updateUserMetrics')
+      await timed(updateNftMetrics, 'CRON updateNftMetrics')
+    })()
+  )
+)
+addEventListener('scheduled', (event) =>
+  event.waitUntil(timed(updateNftDealMetrics, 'CRON updateNftDealMetrics'))
+)
+addEventListener('scheduled', (event) =>
+  event.waitUntil(timed(updatePinStatuses, 'CRON updatePinStatuses'))
+)
```

Comment:
why 3 listeners ?

Reply:
max 3 cron per script according to the docs